File: NBAFullseason/scripts/store.mjs
import { fileURLToPath } from "url";
import { createStore } from "../../core-js/store.mjs";

const DATA_PATH = fileURLToPath(new URL("../data/history.json", import.meta.url));
const { loadStore, saveStore, upsertRun } = createStore(DATA_PATH);

export { loadStore, saveStore, upsertRun };

File: pyFull/scripts/store.py
# scripts/store.py  -- thin wrapper around core/store.py
import sys, os

sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', '..'))

import core.store as _store

_store.DATA_PATH = os.path.abspath(os.path.normpath(
    os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', 'data', 'history.json')
))

# Re-export all public API
from core.store import load_store, save_store, upsert_run
